src/main/scala/scalaz/stream/Process.scala

```diff
@@ -737,8 +753,8 @@ object Process extends ProcessInstances {
   /** The `Process` which emits no values and signals normal termination. */
   val halt: Process0[Nothing] = halt0
 
-  /** Alias for `halt`. */
-  def empty[F[_],O]: Process[F, O] = halt
+  /** Empty emit. Used to signal end of evaluation form `step` to give chance step to be interrupted */
+  val empty: Emit[Nothing] = Emit(Nil)
```

Comment:
@djspiewak yes, it only serves the special purpose in step. I think we may actually use it more generally but I think we can either make this private so users won't be confused or delete it alltogether. 

@fthomas can you please describe where that would be usefull? Otherwise I am more behind what @djspiewak suggesting and just remove it.